model/histogram/float_histogram.go

```diff
@@ -49,35 +50,54 @@ type FloatHistogram struct {
 	// Observation counts in buckets. Each represents an absolute count and
 	// must be zero or positive.
 	PositiveBuckets, NegativeBuckets []float64
+	// Holds the custom (usually upper) bounds for bucket definitions, otherwise nil.
+	// This slice is interned, to be treated as immutable and copied by reference.
+	// These numbers should be strictly increasing. This field is only used when the
+	// schema is for custom buckets, and the ZeroThreshold, ZeroCount, NegativeSpans
+	// and NegativeBuckets fields are not used.
+	CustomValues []float64
```

Comment:
Just curious, why change this field from `CustomBounds` to `CustomValues`?

Reply:
That's due to discussions that happened in #13558 where we realized we can reuse these values for other use cases as well, not just custom buckets.